docs/source/Usage.md

```diff
@@ -134,6 +134,7 @@ The following is the list of the available analyzers you can run out-of-the-box:
 * `DNStwist`: Scan a url/domain to find potentially malicious permutations via dns fuzzing. [dnstwist repo](https://github.com/elceef/dnstwist) 
 * `IPInfo`: Location Information about an IP
 * `Zoomeye`: [Zoomeye](https://www.zoomeye.org) Cyberspace Search Engine recording information of devices, websites, services and components etc..
+* `InQuest`: Deep File Inspection, Indicators of Compromise Database etc...
```

Comment:
could you specify the right name of the new analyzers?

Reply:
You mean `InQuest Labs`?

Reply:
You created 3 different analyzers, I mean those

Reply:
Sure, I just now realized that, I should add this in Advanced Usage, Since it is Optional Analyzer.